### PR TITLE
updated deps

### PR DESCRIPTION
**Pull request by lafin**

### CODE REVIEW

**Comment by vicapow** [comment]
Hey @lafin, thanks for the PR! Before I can merge, can you sign our Contributor License Agreement (CLA)? You can find the [link here](https://docs.google.com/a/uber.com/forms/d/1pAwS_-dA1KhPlfxzYLBqK6rsSWwRwH95OCCZrcsY5rk/viewform). You only need to sign it once for all Uber projects.

**Comment by lafin** [comment]
I accepted

File: package.json [hunk 1]
```diff
@@ -17,22 +17,22 @@
   "license": "MIT",
   "dependencies": {
     "alphaify": "^1.0.0",
-    "bowser": "^0.7.3",
-    "canvas-composite-types": "^1.0.0",
-    "d3": "^3.5.5",
+    "bowser": "^1.0.0",
+    "canvas-composite-types": "^1.0.4",
```
Comment (vicapow) [inline package.json]: I don't think there is a `canvas-composite-types` with version `1.0.4`. This is causing `npm install` to fail:

````lang=sh
npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/Users/victorpowell/.nvm/v0.10.32/bin/npm" "install"
npm ERR! node v0.10.32
npm ERR! npm  v3.7.2

npm ERR! No compatible version found: canvas-composite-types@^1.0.4
npm ERR! Valid install targets:
npm ERR! 1.0.0
````

Comment (vicapow) [inline package.json]: I take that back. Ignore my above comment. ^

**Comment by vicapow** [comment]
Thanks @lafin!

**Pull request merged**